packages/@glimmer/interfaces/lib/managers/internal/modifier.d.ts

```diff
@@ -22,8 +22,8 @@ export interface InternalModifierManager<
   // the modifier's update hooks need to be called (if at all).
   getTag(modifier: TModifierInstanceState): UpdatableTag | null;
 
-  getDebugName(Modifier: TModifierDefinitionState): string;
-  getDebugInstance(Modifier: TModifierInstanceState): unknown;
+  getDebugName(modifier: TModifierInstanceState): string;
```

Comment:
This chance (and everywhere else) is backwards – as I tried to explain previously we want `getDebugName` to be static (takes the definition instead of instance) so we can get the name without constructing it. See the component manager for parallel

Reply:
but the name could depend on the state returned from createModifier

Reply:
done. i think

Reply:
doesn't look like it, did you forget to push or something?

Reply:
all that changed is capitalization